drivers/disk/ramdisk.c

```diff
@@ -12,13 +12,19 @@
 #include <zephyr/init.h>
 #include <zephyr/device.h>
 #include <zephyr/logging/log.h>
+#include "lz4frame.h"
```

Comment:
Should we include it unconditionally to this file? Also should it be `#include ""` instead of `#include <>`?

Reply:
Regarding " vs < it was included with " in a sample for some reason so I assumed that it is required, changed to <